lib/radfiledata.h

```diff
@@ -56,6 +56,7 @@ class STADIC_API RadFileData
     std::shared_ptr<RadPrimitive> addPrimitive(RadPrimitive *primitive);    //!< Add a rad primitive to the list of primitives
     std::shared_ptr<RadPrimitive> addPrimitive(std::shared_ptr<RadPrimitive> primitive);  //!< Add a rad primitive to the list of primitives
     bool setPrimitives(const shared_vector<RadPrimitive> &primitives);  //!< Replace the contents of the primitives list
+    std::shared_ptr<RadPrimitive> RadFileData::glazingsearch(std::string name);
```

Comment:
Now that you've merged in develop, replace any calls to this with

    radData.findMaterial([](std::shared_ptr<stadic::RadPrimitive> p){return p->name() == nameToFind;})

and delete this method.